src/ansys/fluent/core/services/datamodel_tui.py

```diff
@@ -325,14 +325,20 @@ def __init__(self, service, version, mode, path) -> None:
         self._path = path
 
     def __dir__(self) -> list[str]:
-        return [
+        ret_val = [
             convert_tui_menu_to_func_name(x)
             for x in PyMenu(
                 self._service, self._version, self._mode, self._path
             ).get_child_names()
         ]
+        ret_val.remove("exit") if "exit" in ret_val else None
```

Comment:
"exit" could be filtered out in the original expression. If using a separate statement (based on `if` rather than `try`-`except`), it should be in the simple form:
```Python
     if "exit" in ret_val:
         ret_val.remove("exit")
```

Reply:
Done, Thank you.